WebNetSample/Views/Home/Index.cshtml

```diff
@@ -1,9 +1,20 @@
-﻿@{
+﻿@using WebNetSample.Entity.Dtos;
+@model IEnumerable<CategoryDetailDto>
+
+@{
     ViewData["Title"] = "Home Page";
+    Layout = "_HomeLayout";
 }
 
 <div class="text-center">
     <h1 class="display-4">Welcome to Epam WEB .NET Mentoring Program</h1>
-    <h5 class="text-danger"><span class="text-dark">Mentee </span> Osman Ali YARDIM</h5>
-    <h5 class="text-warning"><span class="text-dark">Mentor </span> Dzmitry Yaniuk</h5>
+
+    @section Images {
```

Comment:
I advise you declare all sections and code related to C# context at the beginning of your view code (for example, after the first C# context usage with `ViewData` and `Layout`, rows 4-6). In this case will not have to mix your code declarations (HTML + C#) and it will be easier to read because C# takes the first part and HTML the second

Reply:
But that section needs to be in that div tag? Is it still possible to define where you suggest?

Reply:
Well, the place of declaration != place of usage. If you need to call it here you may just call method `RenderSection`